packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx

```diff
@@ -0,0 +1,7 @@
+import { styled } from '../../styles'
+
+export const StyledButtonGroup = styled('div', {
+  $leva__flex: '',
+  justifyContent: 'flex-end',
+  gap: '$leva__rowGap',
```

Comment:
Sorry I said `rowGap` It should be `colGap` I'm an idiot 😅

Reply:
done :)